### PR TITLE
Renaming all "item" files to "package" files

### PR DESCRIPTION
**Pull request by sdwheeler**
Related to PR #3156  which changed the terminology in the articles but didn't change any file or folder names.

This PR changes the files names, fixes links to new names, and adds redirection from the old name.

### CODE REVIEW

**Comment by zjalexander** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[.openpublishing.redirection.json](https://github.com/sdwheeler/PowerShell-Docs/blob/staging/.openpublishing.redirection.json) | :white_check_mark:Succeeded | |
[gallery/Images/manual_package_download.png](https://github.com/sdwheeler/PowerShell-Docs/blob/staging/gallery/Images/manual_package_download.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/gallery/how-to/finding-packages/searching-by-psedition?branch=pr-en-us-3159) |
[gallery/Images/packagedisplaypagewithpseditions.png](https://github.com/sdwheeler/PowerShell-Docs/blob/staging/gallery/Images/packagedisplaypagewithpseditions.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/gallery/how-to/working-with-packages/manual-download?branch=pr-en-us-3159) |
[gallery/TOC.yml](https://github.com/sdwheeler/PowerShell-Docs/blob/staging/gallery/TOC.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/gallery/concepts/publishing-guidelines?branch=pr-en-us-3159) |
[gallery/bread/toc.yml](https://github.com/sdwheeler/PowerShell-Docs/blob/staging/gallery/bread/toc.yml) | :white_check_mark:Succeeded | |
[gallery/concepts/module-license-acceptance.md](https://github.com/sdwheeler/PowerShell-Docs/blob/staging/gallery/concepts/module-license-acceptance.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/gallery/concepts/module-license-acceptance?branch=pr-en-us-3159) |
[gallery/concepts/module-prerelease-support.md](https://github.com/sdwheeler/PowerShell-Docs/blob/staging/gallery/concepts/module-prerelease-support.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/gallery/concepts/module-prerelease-support?branch=pr-en-us-3159) |
[gallery/concepts/module-psedition-support.md](https://github.com/sdwheeler/PowerShell-Docs/blob/staging/gallery/concepts/module-psedition-support.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/gallery/concepts/module-psedition-support?branch=pr-en-us-3159) |
[gallery/concepts/package-manifest-affecting-ui.md](https://github.com/sdwheeler/PowerShell-Docs/blob/staging/gallery/concepts/package-manifest-affecting-ui.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/gallery/concepts/package-manifest-affecting-ui?branch=pr-en-us-3159) |
[gallery/concepts/publishing-guidelines.md](https://github.com/sdwheeler/PowerShell-Docs/blob/staging/gallery/concepts/publishing-guidelines.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/gallery/concepts/publishing-guidelines?branch=pr-en-us-3159) |
[gallery/concepts/script-license-acceptance.md](https://github.com/sdwheeler/PowerShell-Docs/blob/staging/gallery/concepts/script-license-acceptance.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/gallery/concepts/script-license-acceptance?branch=pr-en-us-3159) |
[gallery/concepts/script-prerelease-support.md](https://github.com/sdwheeler/PowerShell-Docs/blob/staging/gallery/concepts/script-prerelease-support.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/gallery/concepts/script-prerelease-support?branch=pr-en-us-3159) |
[gallery/concepts/script-psedition-support.md](https://github.com/sdwheeler/PowerShell-Docs/blob/staging/gallery/concepts/script-psedition-support.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/gallery/concepts/script-psedition-support?branch=pr-en-us-3159) |
[gallery/concepts/scriptanalyzer-rule-profile.md](https://github.com/sdwheeler/PowerShell-Docs/blob/staging/gallery/concepts/scriptanalyzer-rule-profile.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/gallery/concepts/scriptanalyzer-rule-profile?branch=pr-en-us-3159) |
[gallery/faqs.md](https://github.com/sdwheeler/PowerShell-Docs/blob/staging/gallery/faqs.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/gallery/faqs?branch=pr-en-us-3159) |
[gallery/getting-started.md](https://github.com/sdwheeler/PowerShell-Docs/blob/staging/gallery/getting-started.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/gallery/getting-started?branch=pr-en-us-3159) |
[gallery/how-to/finding-packages/filtering-packages.md](https://github.com/sdwheeler/PowerShell-Docs/blob/staging/gallery/how-to/finding-packages/filtering-packages.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/gallery/how-to/finding-packages/filtering-packages?branch=pr-en-us-3159) |
[gallery/how-to/finding-packages/search-syntax.md](https://github.com/sdwheeler/PowerShell-Docs/blob/staging/gallery/how-to/finding-packages/search-syntax.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/gallery/how-to/finding-packages/search-syntax?branch=pr-en-us-3159) |
[gallery/how-to/finding-packages/searching-by-psedition.md](https://github.com/sdwheeler/PowerShell-Docs/blob/staging/gallery/how-to/finding-packages/searching-by-psedition.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/gallery/how-to/finding-packages/searching-by-psedition?branch=pr-en-us-3159) |
[gallery/how-to/getting-support/bootstrapping-nuget.md](https://github.com/sdwheeler/PowerShell-Docs/blob/staging/gallery/how-to/getting-support/bootstrapping-nuget.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/gallery/how-to/getting-support/bootstrapping-nuget?branch=pr-en-us-3159) |
[gallery/how-to/getting-support/contacting-administrators.md](https://github.com/sdwheeler/PowerShell-Docs/blob/staging/gallery/how-to/getting-support/contacting-administrators.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/gallery/how-to/getting-support/contacting-administrators?branch=pr-en-us-3159) |
[gallery/how-to/getting-support/report-abuse.md](https://github.com/sdwheeler/PowerShell-Docs/blob/staging/gallery/how-to/getting-support/report-abuse.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/gallery/how-to/getting-support/report-abuse?branch=pr-en-us-3159) |
[gallery/how-to/getting-support/troubleshooting-cmdlets.md](https://github.com/sdwheeler/PowerShell-Docs/blob/staging/gallery/how-to/getting-support/troubleshooting-cmdlets.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/gallery/how-to/getting-support/troubleshooting-cmdlets?branch=pr-en-us-3159) |
[gallery/how-to/managing-profile/creating-APIkeys.md](https://github.com/sdwheeler/PowerShell-Docs/blob/staging/gallery/how-to/managing-profile/creating-APIkeys.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/gallery/how-to/managing-profile/creating-apikeys?branch=pr-en-us-3159) |
[gallery/how-to/managing-profile/managing-account.md](https://github.com/sdwheeler/PowerShell-Docs/blob/staging/gallery/how-to/managing-profile/managing-account.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/gallery/how-to/managing-profile/managing-account?branch=pr-en-us-3159) |
[gallery/how-to/publishing-packages/creating-an-account.md](https://github.com/sdwheeler/PowerShell-Docs/blob/staging/gallery/how-to/publishing-packages/creating-an-account.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/gallery/how-to/publishing-packages/creating-an-account?branch=pr-en-us-3159) |
[gallery/how-to/publishing-packages/deleting-packages.md](https://github.com/sdwheeler/PowerShell-Docs/blob/staging/gallery/how-to/publishing-packages/deleting-packages.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/gallery/how-to/publishing-packages/deleting-packages?branch=pr-en-us-3159) |
[gallery/how-to/publishing-packages/managing-package-owners.md](https://github.com/sdwheeler/PowerShell-Docs/blob/staging/gallery/how-to/publishing-packages/managing-package-owners.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/gallery/how-to/publishing-packages/managing-package-owners?branch=pr-en-us-3159) |
[gallery/how-to/publishing-packages/publishing-a-package.md](https://github.com/sdwheeler/PowerShell-Docs/blob/staging/gallery/how-to/publishing-packages/publishing-a-package.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/gallery/how-to/publishing-packages/publishing-a-package?branch=pr-en-us-3159) |
[gallery/how-to/publishing-packages/unlisting-packages.md](https://github.com/sdwheeler/PowerShell-Docs/blob/staging/gallery/how-to/publishing-packages/unlisting-packages.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/gallery/how-to/publishing-packages/unlisting-packages?branch=pr-en-us-3159) |
[gallery/how-to/working-with-packages/contacting-package-owners.md](https://github.com/sdwheeler/PowerShell-Docs/blob/staging/gallery/how-to/working-with-packages/contacting-package-owners.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/gallery/how-to/working-with-packages/contacting-package-owners?branch=pr-en-us-3159) |
[gallery/how-to/working-with-packages/deploy-to-azure-automation.md](https://github.com/sdwheeler/PowerShell-Docs/blob/staging/gallery/how-to/working-with-packages/deploy-to-azure-automation.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/gallery/how-to/working-with-packages/deploy-to-azure-automation?branch=pr-en-us-3159) |
[gallery/how-to/working-with-packages/filelist-view.md](https://github.com/sdwheeler/PowerShell-Docs/blob/staging/gallery/how-to/working-with-packages/filelist-view.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/gallery/how-to/working-with-packages/filelist-view?branch=pr-en-us-3159) |
[gallery/how-to/working-with-packages/manual-download.md](https://github.com/sdwheeler/PowerShell-Docs/blob/staging/gallery/how-to/working-with-packages/manual-download.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/gallery/how-to/working-with-packages/manual-download?branch=pr-en-us-3159) |
[gallery/how-to/working-with-packages/packages-that-require-license-acceptance.md](https://github.com/sdwheeler/PowerShell-Docs/blob/staging/gallery/how-to/working-with-packages/packages-that-require-license-acceptance.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/gallery/how-to/working-with-packages/packages-that-require-license-acceptance?branch=pr-en-us-3159) |
[gallery/how-to/working-with-packages/social-media-feedback.md](https://github.com/sdwheeler/PowerShell-Docs/blob/staging/gallery/how-to/working-with-packages/social-media-feedback.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/gallery/how-to/working-with-packages/social-media-feedback?branch=pr-en-us-3159) |
[gallery/installing-psget.md](https://github.com/sdwheeler/PowerShell-Docs/blob/staging/gallery/installing-psget.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/gallery/installing-psget?branch=pr-en-us-3159) |
[gallery/readme.md](https://github.com/sdwheeler/PowerShell-Docs/blob/staging/gallery/readme.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/gallery/readme?branch=pr-en-us-3159) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C10%5C24%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201810241756418650-3159%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=4ZUTAFVrw6lPckOcvSQqtY3EyZ6UdB3SE6dLxv1UExw%3D&st=2018-10-24T18%3A07%3A45Z&se=2018-11-24T18%3A12%3A45Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: gallery/TOC.yml [hunk 1]
```diff
@@ -9,37 +9,37 @@
...
-          href: how-to/finding-items/searching-by-psedition.md
+          href: how-to/finding-packages/searching-by-psedition.md
         - name: Filtering Search Results
-          href: how-to/finding-items/filtering-items.md
+          href: how-to/finding-packages/filtering-packages.md
     - name: Publishing to the Gallery
       items:
         - name: Creating and publishing items
-          href: how-to/publishing-items/publishing-an-item.md
+          href: how-to/publishing-packages/publishing-a-package.md
```
Comment (DCtheGeek) [inline gallery/TOC.yml]: The files were changed to 'an-package'.  I think this is more correct, but one or the other needs fixing.

File: gallery/concepts/publishing-guidelines.md [hunk 2]
```diff
@@ -17,9 +17,9 @@ improving your code with feedback from initial reviewers and [Powershell Script
 versioning your module, documentation, tests & examples for how to use what you have shared.
 Much of this documentation follows the guidelines for publishing [High Quality DSC Resource Modules](https://github.com/PowerShell/DscResources/blob/master/HighQualityModuleGuidelines.md).
 
-For the mechanics of publishing a package to the PowerShell Gallery, see [Creating and Publishing a Package](https://msdn.microsoft.com/powershell/gallery/psgallery/creating-and-publishing-an-item).
+For the mechanics of publishing a package to the PowerShell Gallery, see [Creating and Publishing a Package](/powershell/gallery/how-to/publishing-packages/publishing-a-package).
```
Comment (DCtheGeek) [inline gallery/concepts/publishing-guidelines.md]: Same issues here.  an vs a in name.

**Comment by zjalexander** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[.openpublishing.redirection.json](https://github.com/sdwheeler/PowerShell-Docs/blob/staging/.openpublishing.redirection.json) | :white_check_mark:Succeeded | |
[gallery/Images/manual_package_download.png](https://github.com/sdwheeler/PowerShell-Docs/blob/staging/gallery/Images/manual_package_download.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/gallery/how-to/finding-packages/searching-by-psedition?branch=pr-en-us-3159) |
[gallery/Images/packagedisplaypagewithpseditions.png](https://github.com/sdwheeler/PowerShell-Docs/blob/staging/gallery/Images/packagedisplaypagewithpseditions.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/gallery/how-to/working-with-packages/manual-download?branch=pr-en-us-3159) |
[gallery/TOC.yml](https://github.com/sdwheeler/PowerShell-Docs/blob/staging/gallery/TOC.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/gallery/concepts/publishing-guidelines?branch=pr-en-us-3159) |
[gallery/bread/toc.yml](https://github.com/sdwheeler/PowerShell-Docs/blob/staging/gallery/bread/toc.yml) | :white_check_mark:Succeeded | |
[gallery/concepts/module-license-acceptance.md](https://github.com/sdwheeler/PowerShell-Docs/blob/staging/gallery/concepts/module-license-acceptance.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/gallery/concepts/module-license-acceptance?branch=pr-en-us-3159) |
[gallery/concepts/module-prerelease-support.md](https://github.com/sdwheeler/PowerShell-Docs/blob/staging/gallery/concepts/module-prerelease-support.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/gallery/concepts/module-prerelease-support?branch=pr-en-us-3159) |
[gallery/concepts/module-psedition-support.md](https://github.com/sdwheeler/PowerShell-Docs/blob/staging/gallery/concepts/module-psedition-support.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/gallery/concepts/module-psedition-support?branch=pr-en-us-3159) |
[gallery/concepts/package-manifest-affecting-ui.md](https://github.com/sdwheeler/PowerShell-Docs/blob/staging/gallery/concepts/package-manifest-affecting-ui.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/gallery/concepts/package-manifest-affecting-ui?branch=pr-en-us-3159) |
[gallery/concepts/publishing-guidelines.md](https://github.com/sdwheeler/PowerShell-Docs/blob/staging/gallery/concepts/publishing-guidelines.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/gallery/concepts/publishing-guidelines?branch=pr-en-us-3159) |
[gallery/concepts/script-license-acceptance.md](https://github.com/sdwheeler/PowerShell-Docs/blob/staging/gallery/concepts/script-license-acceptance.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/gallery/concepts/script-license-acceptance?branch=pr-en-us-3159) |
[gallery/concepts/script-prerelease-support.md](https://github.com/sdwheeler/PowerShell-Docs/blob/staging/gallery/concepts/script-prerelease-support.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/gallery/concepts/script-prerelease-support?branch=pr-en-us-3159) |
[gallery/concepts/script-psedition-support.md](https://github.com/sdwheeler/PowerShell-Docs/blob/staging/gallery/concepts/script-psedition-support.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/gallery/concepts/script-psedition-support?branch=pr-en-us-3159) |
[gallery/concepts/scriptanalyzer-rule-profile.md](https://github.com/sdwheeler/PowerShell-Docs/blob/staging/gallery/concepts/scriptanalyzer-rule-profile.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/gallery/concepts/scriptanalyzer-rule-profile?branch=pr-en-us-3159) |
[gallery/faqs.md](https://github.com/sdwheeler/PowerShell-Docs/blob/staging/gallery/faqs.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/gallery/faqs?branch=pr-en-us-3159) |
[gallery/getting-started.md](https://github.com/sdwheeler/PowerShell-Docs/blob/staging/gallery/getting-started.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/gallery/getting-started?branch=pr-en-us-3159) |
[gallery/how-to/finding-packages/filtering-packages.md](https://github.com/sdwheeler/PowerShell-Docs/blob/staging/gallery/how-to/finding-packages/filtering-packages.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/gallery/how-to/finding-packages/filtering-packages?branch=pr-en-us-3159) |
[gallery/how-to/finding-packages/search-syntax.md](https://github.com/sdwheeler/PowerShell-Docs/blob/staging/gallery/how-to/finding-packages/search-syntax.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/gallery/how-to/finding-packages/search-syntax?branch=pr-en-us-3159) |
[gallery/how-to/finding-packages/searching-by-psedition.md](https://github.com/sdwheeler/PowerShell-Docs/blob/staging/gallery/how-to/finding-packages/searching-by-psedition.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/gallery/how-to/finding-packages/searching-by-psedition?branch=pr-en-us-3159) |
[gallery/how-to/getting-support/bootstrapping-nuget.md](https://github.com/sdwheeler/PowerShell-Docs/blob/staging/gallery/how-to/getting-support/bootstrapping-nuget.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/gallery/how-to/getting-support/bootstrapping-nuget?branch=pr-en-us-3159) |
[gallery/how-to/getting-support/contacting-administrators.md](https://github.com/sdwheeler/PowerShell-Docs/blob/staging/gallery/how-to/getting-support/contacting-administrators.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/gallery/how-to/getting-support/contacting-administrators?branch=pr-en-us-3159) |
[gallery/how-to/getting-support/report-abuse.md](https://github.com/sdwheeler/PowerShell-Docs/blob/staging/gallery/how-to/getting-support/report-abuse.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/gallery/how-to/getting-support/report-abuse?branch=pr-en-us-3159) |
[gallery/how-to/getting-support/troubleshooting-cmdlets.md](https://github.com/sdwheeler/PowerShell-Docs/blob/staging/gallery/how-to/getting-support/troubleshooting-cmdlets.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/gallery/how-to/getting-support/troubleshooting-cmdlets?branch=pr-en-us-3159) |
[gallery/how-to/managing-profile/creating-APIkeys.md](https://github.com/sdwheeler/PowerShell-Docs/blob/staging/gallery/how-to/managing-profile/creating-APIkeys.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/gallery/how-to/managing-profile/creating-apikeys?branch=pr-en-us-3159) |
[gallery/how-to/managing-profile/managing-account.md](https://github.com/sdwheeler/PowerShell-Docs/blob/staging/gallery/how-to/managing-profile/managing-account.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/gallery/how-to/managing-profile/managing-account?branch=pr-en-us-3159) |
[gallery/how-to/publishing-packages/creating-an-account.md](https://github.com/sdwheeler/PowerShell-Docs/blob/staging/gallery/how-to/publishing-packages/creating-an-account.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/gallery/how-to/publishing-packages/creating-an-account?branch=pr-en-us-3159) |
[gallery/how-to/publishing-packages/deleting-packages.md](https://github.com/sdwheeler/PowerShell-Docs/blob/staging/gallery/how-to/publishing-packages/deleting-packages.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/gallery/how-to/publishing-packages/deleting-packages?branch=pr-en-us-3159) |
[gallery/how-to/publishing-packages/managing-package-owners.md](https://github.com/sdwheeler/PowerShell-Docs/blob/staging/gallery/how-to/publishing-packages/managing-package-owners.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/gallery/how-to/publishing-packages/managing-package-owners?branch=pr-en-us-3159) |
[gallery/how-to/publishing-packages/publishing-a-package.md](https://github.com/sdwheeler/PowerShell-Docs/blob/staging/gallery/how-to/publishing-packages/publishing-a-package.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/gallery/how-to/publishing-packages/publishing-a-package?branch=pr-en-us-3159) |
[gallery/how-to/publishing-packages/unlisting-packages.md](https://github.com/sdwheeler/PowerShell-Docs/blob/staging/gallery/how-to/publishing-packages/unlisting-packages.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/gallery/how-to/publishing-packages/unlisting-packages?branch=pr-en-us-3159) |
[gallery/how-to/working-with-packages/contacting-package-owners.md](https://github.com/sdwheeler/PowerShell-Docs/blob/staging/gallery/how-to/working-with-packages/contacting-package-owners.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/gallery/how-to/working-with-packages/contacting-package-owners?branch=pr-en-us-3159) |
[gallery/how-to/working-with-packages/deploy-to-azure-automation.md](https://github.com/sdwheeler/PowerShell-Docs/blob/staging/gallery/how-to/working-with-packages/deploy-to-azure-automation.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/gallery/how-to/working-with-packages/deploy-to-azure-automation?branch=pr-en-us-3159) |
[gallery/how-to/working-with-packages/filelist-view.md](https://github.com/sdwheeler/PowerShell-Docs/blob/staging/gallery/how-to/working-with-packages/filelist-view.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/gallery/how-to/working-with-packages/filelist-view?branch=pr-en-us-3159) |
[gallery/how-to/working-with-packages/manual-download.md](https://github.com/sdwheeler/PowerShell-Docs/blob/staging/gallery/how-to/working-with-packages/manual-download.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/gallery/how-to/working-with-packages/manual-download?branch=pr-en-us-3159) |
[gallery/how-to/working-with-packages/packages-that-require-license-acceptance.md](https://github.com/sdwheeler/PowerShell-Docs/blob/staging/gallery/how-to/working-with-packages/packages-that-require-license-acceptance.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/gallery/how-to/working-with-packages/packages-that-require-license-acceptance?branch=pr-en-us-3159) |
[gallery/how-to/working-with-packages/social-media-feedback.md](https://github.com/sdwheeler/PowerShell-Docs/blob/staging/gallery/how-to/working-with-packages/social-media-feedback.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/gallery/how-to/working-with-packages/social-media-feedback?branch=pr-en-us-3159) |
[gallery/installing-psget.md](https://github.com/sdwheeler/PowerShell-Docs/blob/staging/gallery/installing-psget.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/gallery/installing-psget?branch=pr-en-us-3159) |
[gallery/readme.md](https://github.com/sdwheeler/PowerShell-Docs/blob/staging/gallery/readme.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/gallery/readme?branch=pr-en-us-3159) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C10%5C24%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201810242124579900-3159%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=xqftIMykrAE9YUPLC6tzDgYHxzw4jBeffmQD6OVWdT4%3D&st=2018-10-24T21%3A35%3A14Z&se=2018-11-24T21%3A40%3A14Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: gallery/TOC.yml (hunk 1 above)
Comment (sdwheeler) [reply gallery/TOC.yml]: fixed

File: gallery/concepts/publishing-guidelines.md (hunk 2 above)
Comment (sdwheeler) [reply gallery/concepts/publishing-guidelines.md]: no fix needed

**Pull request merged**